### DISCUSSION TITLE
https://XXX.XXXX.XXX" is not working. Is it a pool? Error: Failed to fetch

### DISCUSSION THREAD

**Discussion by Sakuya-CN** [Chia-Network/chia-blockchain#12320] [Support]
I have three devices, all of which have joined the mining pool, but one of the mining pools has a normal link and is running normally, while the other two are abnormally linked, and it prompts "The pool URL "https://asia1.pool.space" is not working. Is it a pool? Error: Failed to fetch", which caused my two machines to have no income for a long time, I don't know what happened

**Comment by Sakuya-CN** [comment]
![0c9235025d5fce3089c9863cfcabb9d](https://user-images.githubusercontent.com/56291932/178428433-01bf1b35-046a-4c60-bab7-88e396514b57.png)
![6ee35a79c4485501399342bbc4c1144](https://user-images.githubusercontent.com/56291932/178428451-0b8576be-fbab-4469-b9d6-d5ac51b48bb5.png)
Here is a screenshot of the software part of the two devices

**Comment by loppefaaret** [comment]
make sure your machine is able to resolve the URL - perhaps try pinging `asia1.pool.space` - ensure you DNS settings are correctly set etc.
